src/Button.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Button = props => {
+  const { numbers, symbols } = props;
+  const numberList = numbers.map(number => <button type="button" key={number.toString()}>{number}</button>);
+  const symbolList = symbols.map(symbol => <button type="button" key={symbol.toString()}>{symbol}</button>);
+
+  return (
+    <div>
+      <div>{numberList}</div>
+      <div>{symbolList}</div>
+    </div>
+  );
+};
```

Comment:
- [ ] don't map the symbols, instead, you should get **one** symbol as a prop that gets rendered as a `<button>` tag